data/human/escorts.txt

```diff
@@ -0,0 +1,19 @@
+#Example of an escort definition:
+#
+#mission "Escort"
+#	name "Escort"
+#	description "This Captain will Escort you for 200 credits per day."
+#	repeat
+#	job
+#	escortCost 200
+#	source
+#		government "Republic" "Free Worlds" "Syndicate" "Neutral" "Independent"
+#	to complete
+#		never
+#	npc accompany save
+#		government "Merchant"
+#		personality escort heroic
+#		fleet
+#			names "civilian"
+#			variant
+#				"Interceptor"
```

Comment:
```suggestion
#mission "Escort"
#	name "Escort"
#	description "This Captain will Escort you for 200 credits per day."
#	repeat
#	job
#	"escort salary" 200
#	source
#		government "Republic" "Free Worlds" "Syndicate" "Neutral" "Independent"
#	to complete
#		never
#	npc accompany save
#		government "Merchant"
#		personality escort heroic
#		fleet
#			names "civilian"
#			variant
#				"Sparrow"
```
This should be moved into PR description but since its here, let me fix it first.

Reply:
Under variant you are supposed to specify the ship type, you can't just type Interceptor there though
There is a wiki (on the bar up top with pull request etc, third one from the right, you can easily find syntax and all in there)